File: Shared/Cdc/Enumerations.cs
namespace TrRebootTools.Shared.Cdc
{
    public enum CdcGame
    {
        Tr2013 = 9,
        Rise,
        Shadow
    }

    public enum ResourceType
    {
        Unknown = 0,
        Animation = 2,
        PsdRes = 4,
        Texture = 5,
        SoundBank = 6,
        Dtp = 7,
        Script = 8,
        ShaderLib = 9,
        Material = 10,
        GlobalContentReference = 11,
        Model = 12,
        CollisionMesh = 13,
        ObjectReference = 14,
        AnimationLib = 15
    }

    public enum ResourceSubType
    {
        Texture = 5,
        Model = 26,
        ModelData = 27,
        ShResource = 116,
        CubeLut = 117
    }
}

File: addons/io_scene_tr_reboot/tr/Enumerations.py
from enum import IntEnum

class CdcGame(IntEnum):
    TR2013 = 9
    ROTTR = 10
    SOTTR = 11

class ResourceType(IntEnum):
    ANIMATION = 2
    PSDRES = 4
    TEXTURE = 5
    SOUND = 6
    DTP = 7
    SCRIPT = 8
    SHADERLIB = 9
    MATERIAL = 10
    GLOBALCONTENTREFERENCE = 11
    MODEL = 12
    COLLISIONMESH = 13
    OBJECTREFERENCE = 14
    TRIGGER = 15
